scrapy_autoextract/providers.py

```diff
@@ -66,11 +71,20 @@ def get_concurrent_requests_per_domain(settings: Settings):
     return concurrency
 
 
+@attr.define()
+class AERequestSpec:
+    query: List[AutoExtractRequest]
```

Comment:
As discussed offline, this was because https://github.com/zytedata/zyte-autoextract is using [Batching API Queries](https://docs.zyte.com/automatic-extraction.html#batching-queries).